apps/test-editor/src/styles.css

```
@@ -0,0 +1,2 @@
+@import '@notopia-uit/ui/styles';
+@source 'src/**/*.{ts,tsx}';
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)